pmm-app/src/pmm-settings/components/AlertManager/AlertManager.tsx

```diff
@@ -65,11 +67,11 @@ export const AlertManager: FC<AlertManagerProps> = ({
           icon="info-circle"
         />
       </div>
-      <input
-        className={cx(settingsStyles.input, styles.input)}
+      <Input
```

Comment:
I think that it makes sense to wrap it with the final form interface. if we do it we simplify interaction and will have the possibility to replace grafana elements with something different if we ever need it

Reply:
Basically do the same for this component that was done for the others? Use the Form from final form etc?

Reply:
Done 👍